### DISCUSSION TITLE
新窗口打开的那种网站，打包之后点击就没有反应。

### DISCUSSION THREAD

**Discussion by kayshen0817** [tw93/Pake#754] [General]
打包了小满和几个其他的网站。发现只要是那种点击之后新标签窗口打开的软件就会没有反应，不会跳转过去。请问是我打包的时候设置的不对吗？

**Comment by tw93** [comment]
试试 pake-cli@2.5.2 版本，此外可以把url贴出来，大伙试试看

**Reply by kayshen0817** [reply]
就是阿里的小满，crm.xiaoman.cn，还有知识星球https://wx.zsxq.com，这类网站，只要是这类点击之后在新的标签中打开的网页，那pake打包好的软件就不会跳转或者新开一个窗口打开，就是无反应状态。我不是程序员，可能描述的不是非常准确，就是点击完就没有反应了。但是只要是那种就在本标签跳转的网页，打包好的应用就可以跳转到目标页面。但是我是在线打包的，也不知道在哪里能够调试。

**Comment by zxying3** [comment]
我使用pack在线打包也是存在这种问题，请问楼主有解决吗

**Reply by kayshen0817** [reply]
我也是在线打包的，也还没解决呢。

**Reply by kayshen0817** [reply]
我现在的解决办法是，有的网站支持右键新窗口打开，然后新窗口是一个webview2的窗口，后面一直用这个窗口运行软件就能不受限制新开窗口打开了。我是因为不喜欢PWA应用老是专门又打开浏览器标签浏览，然后找到了tw93大佬制作的这个pake，但是我又不怎么懂代码，现在这样勉强OK，反正不会打开浏览器了，先用这个吧。T T
